instrumentation/netty/netty-4.1/library/src/main/java/io/opentelemetry/instrumentation/netty/v4_1/internal/server/HttpServerRequestTracingHandler.java

```diff
@@ -66,7 +62,7 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
       // the span is ended normally in HttpServerResponseTracingHandler
     } catch (Throwable throwable) {
       // make sure to remove the server context on end() call
-      ServerContext serverContext = serverContexts.removeLast();
+      ServerContext serverContext = serverContexts.pollLast();
       instrumenter.end(serverContext.context(), serverContext.request(), null, throwable);
```

Comment:
do we need to null check `serverContext` with the change from `removeLast` to `pollLast`?

Reply:
good catch, thanks